chain/network/src/types.rs

```diff
@@ -287,6 +319,7 @@ pub struct NetworkInfo {
     /// Accounts of known block and chunk producers from routing table.
     pub known_producers: Vec<KnownProducer>,
```

Comment:
Wait, what's this known_producer thing?  From the comment, it sounds suspiciously like tier1, no?

Reply:
Known producers is a list of accounts, yes, however
- it doesn't contain endpoints, just PeerId 
- it distributes ALL accounts, no matter if they are active validators, therefore it doesn't support resynchronizing in case the set of important accounts changes 
- it uses LRU cache to keep this information, not giving a damn about which accounts are actually important
- it doesn't support sub-epoch versioning of the received data
- it doesn't implement any abuse-prevention mechanisms.